queries/cairo/indents.scm

```diff
@@ -0,0 +1,44 @@
+[
+  (import_statement)
```

Comment:
I don't think this should be here.

If it should, then restricting to `(import_statement (. "from"))` or it having braces/parenthesis will be better

Reply:
yeah you're right it needs a parenthesis